core/src/main/java/com/novoda/merlin/MerlinsBeard.java

```diff
@@ -10,7 +11,8 @@
  */
 public class MerlinsBeard {
 
-    private ConnectivityManager connectivityManager;
+    private final ConnectivityManager connectivityManager;
+    private final TelephonyManager telephonyManager;
```

Comment:
Mmm I don't think so, at least not for the check we are doing.
The demo app is working correctly without any new permission..